remote-scripts/dpdk_bindadapter_2.sh

```diff
@@ -0,0 +1,131 @@
+#!/bin/bash
+
+###############################################################################
+##
+## Description:
+##  Bind Network Adapter to DPDK driver
+##
+###############################################################################
+##
+## Revision:
+## v1.0 - Ruowen Qin - 7/5/2018 - Build the script.
+##
+###############################################################################
+
+: '
+        <test>
+            <testName>dpdk_bindadapter</testName>
+            <testID>ESX-DPDK-002</testID>
+            <testScript>dpdk_bindadapter_2.sh</testScript>
+            <files>remote-scripts/dpdk_bindadapter_2.sh</files>
+            <files>remote-scripts/utils.sh</files>
+            <RevertDefaultSnapshot>False</RevertDefaultSnapshot>
+            <testParams>
+                <param>TC_COVERED=RHEL-136145</param>
+            </testParams>
+            <timeout>240</timeout>
+            <onError>Continue</onError>
+            <noReboot>True</noReboot>
+        </test>
+
+'
+
+
+dos2unix utils.sh
+
+#
+# Source utils.sh
+#
+. utils.sh || {
+    echo "Error: unable to source utils.sh!"
+    exit 1
+}
+
+#
+# Source constants file and initialize most common variables
+#
+UtilsInit
+
+###############################################################################
+##
+## Put your test script here
+## NOTES:
+## 1. Please use LogMsg to output log to terminal.
+## 2. Please use UpdateSummary to output log to summary.log file.
+## 3. Please use SetTestStateFailed, SetTestStateAborted, SetTestStateCompleted,
+##    and SetTestStateRunning to mark test status.
+##
+###############################################################################
+
+
+GetDistro
+
+if [ "$DISTRO" == "redhat_6" ]
+then
+    SetTestStateSkipped
+    LogMsg "Not support rhel6"
+    exit 1
+fi
+
+
+# This will bind second and thrid network adapter to DPDK
+source /etc/profile.d/dpdk.sh
+
```

Comment:
1. Where is dpdk.sh from? After DPSK source code is compliked?

Reply:
Yeah have to be in sequence compile, bind, helloworld, KNI